docs/how-to-docs/reloading-certificates-within-sidecar.md

```diff
@@ -0,0 +1,73 @@
+##Using Cert Manager with an Nginx sidecar
+
+
+If you are using Nginx sidecar to terminate TLS in pods then you will need to make some updates to your deployment and add a new Certificate when migrating over to Cert Manager.
+
+This can be achieved in three steps.
+1.Creating a new certificate.
+2.Updating your deployment.
```

Comment:
3. below is part of the deployment too, so I suggest we replace 2 and 3 with:
2. Updating your deployment to load use the new certificate and periodically reload it